toolboxes/fluid/cases/lid-driven-cavity/cfd2d.cfg

```diff
@@ -4,6 +4,10 @@ directory=toolboxes/fluid/lid-driven-cavity/2d/P2P1G1
 case.dimension=2
 #case.discretization=P1P1G1
 
+[nirb]
```

Comment:
this file should not be modified! please revert

Reply:
done in 941376ed5a8dda927b1888a051e6d760103df348